Automattic/wp-calypso
# Update comment counts

**Built on top of https://github.com/Automattic/wp-calypso/pull/40720**

#### Changes proposed in this Pull Request

Update realtime comment counts
- In following view http://calypso.localhost:3000/read
- In feed view http://calypso.localhost:3000/read/feeds/<feedId> you can get there by clicking the author name in the following view
- In full post view for both feed and blog http://calypso.localhost:3000/read/blogs/<id>/posts/<id> by clicking the post title in a feed/conversation

#### Testing instructions
Example https://d.pr/v/euB0QQ

- Open all the above pages
- Apply patch D41517
- Sandbox one of your blogs, follow it and start a conversation (multiple comments and replies)
- Add a comment to one of your posts on the frontend, the comment count should increase in calypso
- Try the same after hard refreshing the section
